src/main/steps/applicant1/review-the-application/content.ts

```diff
@@ -171,6 +172,7 @@ const languages = {
 
 export const generateContent: TranslationFn = (content: CommonContent) => {
   const { language, userCase } = content;
+  content.partner = getApplicant1PartnerContent(content);
```

Comment:
if you changed the partner here then all calls to ${partner} on the page will be referencing the Applicant1Partner which I don't think is correct. It's probably better to call this 

  `content.applicant1Partner = getApplicant1PartnerContent(content);`

and then update that specific partner mentioned in the ticket to be ${applicant1Partner}

Reply:
Changed the value of partner to applicant1Partner.

Reply:
All done.